src/cryptonote_config.h

```diff
@@ -54,16 +54,26 @@
 #define EMISSION_SPEED_FACTOR_PER_MINUTE                (20)
 #define FINAL_SUBSIDY_PER_MINUTE                        ((uint64_t)30) // 3 * pow(10, 1)
 
-#define CRYPTONOTE_REWARD_BLOCKS_WINDOW                 100
-#define CRYPTONOTE_BLOCK_GRANTED_FULL_REWARD_ZONE_V2    60000 //size of block (bytes) after which reward for block calculated using block size
-#define CRYPTONOTE_BLOCK_GRANTED_FULL_REWARD_ZONE_V1    20000 //size of block (bytes) after which reward for block calculated using block size - before first fork
-#define CRYPTONOTE_BLOCK_GRANTED_FULL_REWARD_ZONE_V5    300000 //size of block (bytes) after which reward for block calculated using block size - second change, from v5
+#define CRYPTONOTE_REWARD_BLOCKS_WINDOW                     100
+#define CRYPTONOTE_BLOCK_GRANTED_FULL_REWARD_ZONE_V2_OLD    60000 //size of block (bytes) after which reward for block calculated using block size
```

Comment:
You don't need to maintain the old config values in a new "_OLD" variable. Version control is already handled by git.

Keep it simple. :)

Reply:
Yes I included them for personal usage and forgot to remove them for the PR, my bad.

I will be working on the other stuff together with @Starmute later the day.

Reply:
Great! Thank you.